contracts/StakeManagerV2.sol

```diff
@@ -253,6 +285,66 @@ contract StakeManagerV2 is
         emit Redelegated(_fromOperator, _toOperator, _amount);
     }
 
+    /// @notice Undelegate all BNB from all operators.
+    /// @dev This function undelegates all BNB from all registered operators and records
+    ///      the total BNB undelegated and total BNBx supply at the moment of execution.
+    /// @dev This function can only be called by an address with the MANAGER_ROLE.
```

Comment:
lets simulate undelegateAllBnbFromAllOperators to check if it reverts in case operator has lower than some min threshold stake